link-all.sh

```diff
@@ -25,8 +25,10 @@ for module in ${modules}; do
   # Symlink executable scripts into place as well. This is not completely
   # according to spec (we look in the bin/ directory instead of the { "scripts"
   # } entry in package.json but it's quite a bit easier.
-  [[ -d $module/bin ]] && for script in $(find $module/bin -perm +111); do
-    echo "${script} => node_modules/.bin/$(basename $script)"
-    ln -fs ${script} node_modules/.bin
-  done
+  if [[ -d $module/bin ]]; then
+    for script in $(find $module/bin -perm /111); do
```

Comment:
Note on this change (`-perm +111` to `-perm /111`).  The syntax `-perm +omode` was removed in findutils 4.5.12, in favour of `-perm /omode`.  Ubuntu 18.04 has findutils 4.6.0. AL2 has findutils 4.5.11, which provides this warning:
`The interpretation of -perm +omode changed in findutils-4.5.11` and returns no values.